README.md

```diff
@@ -105,7 +102,13 @@ data:
 
 This will use the scanned data to create the high-level conclusions Pulse displays to users and makes available for download.
 
-* Review the changes, rebuild the site, and if all looks good, commit them to source control.
+* Upload Pulse's data to S3 using the `aws` command line tool. Tested on version `1.10.0` of the `awscli` package in `pip`.
+
+```bash
+aws s3 sync data/output/processed/ s3://pulse.cio.gov/live/data/ --acl=public-read
+
+today=$(date +%Y-%m-%d) aws s3 sync data/output/processed/ s3://pulse.cio.gov/$today/data/ --acl=public-read
```

Comment:
Note that I moved away from `s3cmd` to the [official AWS CLI client](http://docs.aws.amazon.com/cli/latest/reference/s3/index.html), which is the `awscli` module in `pip`. The inability of the `s3cmd` module to do proper MIME type detection finally made me angry enough to switch. It was easy, and the tool is much more actively supported, so I recommend that the analytics.usa.gov repo switch to this method as well for any S3 bucket operations.